app/services/reports/adult_incontinence_report_service.rb

```diff
@@ -1,98 +1,97 @@
 module Reports
   class AdultIncontinenceReportService
-    ADULT_INCONTINENCE_TYPES = %w[adult_incontinence underpads pads liners].freeze
+    include ActionView::Helpers::NumberHelper
     attr_reader :year, :organization
 
+    # @param year [Integer]
+    # @param organization [Organization]
     def initialize(year:, organization:)
       @year = year
       @organization = organization
     end
 
+    # @return [Hash]
     def report
-      @report ||= {
-        money_spent: money_spent,
-        monthly_adult_incontinence: monthly_adult_incontinence,
-        provided_per_person: provided_per_person,
-        supplies_distributed: supplies_distributed,
-        supplies_purchased: supplies_purchased,
-        supplies_received: supplies_received
-      }
+      @report ||= { name: 'Adult Incontinence',
+                    entries: {
+                      'Adult incontinence supplies distributed' => number_with_delimiter(distributed_supplies),
+                      'Adult incontinence supplies per adult per month' => monthly_supplies&.round || 0,
+                      'Adult incontinence supplies' => types_of_supplies,
+                      '% adult incontinence supplies donated' => "#{percent_donated.round}%",
+                      '% adult incontinence bought' => "#{percent_bought.round}%",
+                      'Money spent purchasing adult incontinence supplies' => number_to_currency(money_spent_on_supplies)
+                    } }
     end
 
-    def columns_for_csv
-      %i[supplies_distributed monthly_adult_incontinence
-         supplies_received supplies_purchased]
+    # @return [Integer]
+    def distributed_supplies
+      @distributed_supplies ||= organization
+                                .distributions
+                                .for_year(year)
+                                .joins(line_items: :item)
+                                .merge(Item.adult_incontinence)
+                                .sum('line_items.quantity')
     end
 
-    def adult_incontinence_items
-      organization.items.where(partner_key: ADULT_INCONTINENCE_TYPES)
+    # @return [Integer]
+    def monthly_supplies
+      # NOTE: This is asking "per adult per month" but there doesn't seem to be much difference
+      # in calculating per month or per any other time frame, since all it's really asking
+      # is the value of the `distribution_quantity` field for the items we're giving out.
+      organization
+        .distributions
+        .for_year(year)
+        .joins(line_items: :item)
+        .merge(Item.adult_incontinence)
+        .average('items.distribution_quantity')
```

Comment:
Looking at this again, I wonder if we'll have an issue for the orgs that don't set a `distribution_quantity` I wonder if we should use the `default_quantity` method on items instead.

Reply:
I can definitely use a COALESCE to handle those cases.

Reply:
That's some good sql-fu :)

Reply:
Pushed the fix for this!